client/ts/src/market.ts

```diff
@@ -261,6 +261,42 @@ export class Market {
     return this.data.asks;
   }
 
+  /**
+   * Get the most competitive bid price
+   *
+   * @returns number | undefined
+   */
+  public bestBidPrice(): number | undefined {
+    return this.data.bids[this.data.bids.length - 1]?.tokenPrice;
```

Comment:
should we sort the data in the other direction when we deserialize?

Reply:
I don't have a strong opinion. I kind of like leaving the data structure as is in case someone really cares about performance (we are in typescript... so meh? but still?). So they can use the convenience functions to reverse the arrays or just deal with the order when they really don't want any extra fluff.